index.js

```diff
@@ -214,6 +214,14 @@ function showSearch() {
     });
 }
 
+/*
+ * Opens the bookmark menu
+ */
+function showBookmarks() {
+    let activeWindow = windowUtils.getMostRecentBrowserWindow();
+    UITour.showMenu(activeWindow, 'bookmarks');
```

Comment:
Is it safe to simply call this? Our other use wraps this in a `try/catch`, perhaps we can reuse `highlight`. Something like:

```
function uiTourHelper(item, action) {
    if (item === 'urlbar') {
        awesomeBar.addEventListener('focus', removeHighlight);
    }

    UITour.getTarget(activeWindow, item, false).then(function(chosenItem) {
        try {
            if (action === 'highlight') {
                UITour.showHighlight(activeWindow, chosenItem, 'wobble');
            } else if (action === 'show') {
               UITour.showMenu(activeWindow, item);
            }
        } catch(e) {
            console.error('Could not highlight element. Check if UITour.jsm supports highlighting of element passed.', e);
        }
    });
}
```
You would then call it:

```
uiTourHelper('bookmarks', 'show');
```
or
```
uiTourHelper('bookmarks', 'highlight');
```